src/main/kotlin/earth/terrarium/cloche/target/fabric/FabricTargetImpl.kt

```diff
@@ -305,7 +311,7 @@ internal abstract class FabricTargetImpl @Inject constructor(name: String) :
         val commonClasspath = list(
             registerCompilationTransformations(
```

Comment:
Should change the `maybeRegister` to `register` in https://github.com/terrarium-earth/cloche/blob/main/src/main/kotlin/earth/terrarium/cloche/target/TargetCompilation.kt#L45 and https://github.com/terrarium-earth/cloche/blob/main/src/main/kotlin/earth/terrarium/cloche/target/TargetCompilation.kt#L63

Reply:
Forget to push. I'm waiting for the "correct task name" before XD